rocketpool-cli/commands/pdao/signalling-address.go

```diff
@@ -15,20 +21,54 @@ func setSignallingAddress(c *cli.Context, signallingAddress common.Address, sign
 	// // Build the TX
 	// response, err := rp.Api.PDao.SetSignallingAddress()
 
+	// Test Strings
+	fmt.Printf("Signalling Address: %s\n", signallingAddress)
+	fmt.Printf("Signature: %s\n", signature)
+	fmt.Println()
+
+	sig, err := utils.ParseEIP712(signature)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("EIP712Components:")
```

Comment:
This should _absolutely_ by a receiver on sig.

That is, write a:
`func (e *EIP712Components) String() string {` function to basically just do
`return fmt.Sprintf("%+v", e)`

But, critically, the logic that pretty-prints the struct does not belong in signalling-address.go, it belongs along side the definition of EIP712Components.